INSTALL.md

```diff
@@ -70,6 +70,7 @@ The local development configuration uses sqlite3, so no database setup is requir
 
 Run in the /target directory:
 * `python manage.py migrate`
+* `python manage.py loaddata prefencesets` 
```

Comment:
Should this be added to entrypoint.sh for docker startup as well?

Reply:
Added; I wasn't sure of the right point to add it to the container build.